src/timesignature.ts

```diff
@@ -18,18 +18,18 @@ export interface TimeSignatureInfo {
 }
 
 const assertIsValidFraction = (timeSpec: string) => {
-  const numbers = timeSpec.split('/').filter((number) => number !== '');
+  const numbers = timeSpec.split('/');
 
-  if (numbers.length !== 2) {
+  if (numbers.length !== 2 && numbers[0] !== '+') {
     throw new RuntimeError(
       'BadTimeSignature',
       `Invalid time spec: ${timeSpec}. Must be in the form "<numerator>/<denominator>"`
     );
   }
 
   numbers.forEach((number) => {
-    if (isNaN(Number(number))) {
-      throw new RuntimeError('BadTimeSignature', `Invalid time spec: ${timeSpec}. Must contain two valid numbers.`);
+    if (/^([0-9+()]{1,})$/.test(number) == false) {
```

Comment:
Can you add a comment here that makes the format explicit. (Better still, I'd suggest not using a regex for this simple case.)

Reply:
How would you make the test then? Comment would be:
"One or more characters consisting in number 0..9, '+', '(' or ')'"

Reply:
That comment is fine. Maybe simpler regex: `/^[0-9+()]+$/` for the filter.

But for a valid signature, I'd imagine, you need two tests:

* without parens: `/^[0-9]+(\+[0-9]+)?`
* open paren must close: `/^\([0-9]+(\+[0-9]+)?\)\)$/`

Reply:
I will go for the simple one, if someone does not close the parenthesis, they will see that in the results.